examples/jupyter/QuickStart.ipynb

```diff
@@ -0,0 +1,542 @@
+{
```

Comment:
The notebook still need to be rendered to the Readthedocs page. This requires a few changes, as done in https://github.com/ComPWA/tensorwaves/commit/b346f5e7d913ec0bd1065a0d19cca298483af861 (links to the specific changes below):

1. Add `nbsphinx` to the `doc/requirements.txt` (see [here](https://github.com/ComPWA/tensorwaves/commit/b346f5e7d913ec0bd1065a0d19cca298483af861#diff-46507bcc13aee955f6fe861c30ea5444R4)). This Sphinx extension renders Jupyter notebooks.
2. Add a subprocess to the `conf.py` file that Sphinx uses, so that the notebooks are copied from the `examples` folder to `doc` (see diff [here](https://github.com/ComPWA/tensorwaves/commit/b346f5e7d913ec0bd1065a0d19cca298483af861#diff-684a0ceb2878d48b19612ca360e277ef), all changes there are required)
3. Create a page that includes the notebooks that the subprocess copies to `doc/usage` (see [usage.rst](https://github.com/ComPWA/tensorwaves/commit/b346f5e7d913ec0bd1065a0d19cca298483af861#diff-665ffb7d19093ce3b903440e23eb6e98)) and add that page to the `toctree` in the `index.rst` file (see [here](https://github.com/ComPWA/tensorwaves/commit/b346f5e7d913ec0bd1065a0d19cca298483af861#diff-0ced3dfd44ba156272e6db34a5247462R74))
4. Adapt the Travis CI documentation job (see [here](https://github.com/ComPWA/tensorwaves/commit/b346f5e7d913ec0bd1065a0d19cca298483af861#diff-354f30a63fb0907d4ad57269548329e3))

After implementing these changes, you can run:
```bash
cd doc
NBSPHINX_EXECUTE='' make html
```
to see if it all works (setting `NBSPHINX_EXECUTE` is just a trick we use to run the notebook).

Reply:
Please also add `jupyter` to the `doc/requirements.txt` (like [here](https://github.com/ComPWA/tensorwaves/commit/b346f5e7d913ec0bd1065a0d19cca298483af861#diff-46507bcc13aee955f6fe861c30ea5444R2))